Makefile

```diff
@@ -1,43 +1,47 @@
 GO_LINKER_SYMBOL := "github.com/heroku/shh.version"
+GO_BUILD_ENV := GOOS=linux GOARCH=amd64
 
 all: test
 
 test:
-	go test ./...
+	go test -v ./...
+	go test -v -race ./...
 
 install: glv
-	go install -a -ldflags "-X ${GO_LINKER_SYMBOL}=${GO_LINKER_VALUE}" ./...
+	go install -v ${LDFLAGS} ./...
 
-update-deps: godep
+update-deps: govendor
 	godep save -r ./...
```

Comment:
Should this be some govendor command?

Reply:
IMO, no as you shouldn't just blindly update all deps.